### PR TITLE
Rodri/invoices api

### PR DESCRIPTION
**Pull request by RodrigoMolinaMadrid**
The router.ex has been modified in order to add the JSON api. And the module InvoicesController has been created in order to recive the HTTP request of the JSON api, this modue.

### CODE REVIEW

**Review by ale1311sv — APPROVED** [review]

File: lib/siwapp_web/templates/page/index.html.heex [hunk 1]
```diff
@@ -38,4 +38,5 @@
       </li>
     </ul>
   </article>
-</section>
+
+</section>
```
Comment (peillis) [inline lib/siwapp_web/templates/page/index.html.heex:42]: why this change?

File: lib/siwapp_web/controllers/api/invoices_controller.ex [hunk 2]
```diff
@@ -0,0 +1,64 @@
+defmodule SiwappWeb.InvoicesController do
+  use SiwappWeb, :controller
+  # alias AlejandroModuleForRecordings
```
Comment (peillis) [inline lib/siwapp_web/controllers/api/invoices_controller.ex:3]: you'll put this when needed, right now is just rubbish. I would remove all these comments

**Review by peillis — CHANGES_REQUESTED** [review]

**Comment by peillis** [comment]
solve conflicts in route and I'll merge

**Review by peillis — APPROVED** [review]
